rubin_sim/maf/runComparison/radarPlot.py

```diff
@@ -0,0 +1,137 @@
+import numpy as np
+import matplotlib.pylab as plt
+
+# Starting with example at
+# https://matplotlib.org/examples/api/radar_chart.html
+
+from matplotlib.path import Path
+from matplotlib.spines import Spine
+from matplotlib.projections.polar import PolarAxes
+from matplotlib.projections import register_projection
+
+__all__ = ['radar_factory', 'unit_poly_verts', 'radar']
+
+
+def radar_factory(num_vars, frame='circle'):
+    """Create a radar chart with `num_vars` axes.
+
+    This function creates a RadarAxes projection and registers it.
+
+    Parameters
+    ----------
+    num_vars : int
+        Number of variables for radar chart.
+    frame : {'circle' | 'polygon'}
+        Shape of frame surrounding axes.
+
+    """
+    # calculate evenly-spaced axis angles
+    theta = np.linspace(0, 2*np.pi, num_vars, endpoint=False)
+    # rotate theta such that the first axis is at the top, then make sure we don't go past 360
+    theta += np.pi/2
+    theta = theta % (2.*np.pi)
+
+    def draw_poly_patch(self):
+        verts = unit_poly_verts(theta)
+        return plt.Polygon(verts, closed=True, edgecolor='k')
+
+    def draw_circle_patch(self):
+        # unit circle centered on (0.5, 0.5)
+        return plt.Circle((0.5, 0.5), 0.5)
+
+    patch_dict = {'polygon': draw_poly_patch, 'circle': draw_circle_patch}
+    if frame not in patch_dict:
+        raise ValueError('unknown value for `frame`: %s' % frame)
+
+    class RadarAxes(PolarAxes):
```

Comment:
Why do you declare the class within the factory? Doing it this way creates a new class each time it is run, rather than making many instances of the same class. If you put the class on the "outside", you can make the factory a class method, and only have one new class, and avoid having a separate class (but all with the same name) for every time you use the factory.